libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/listener/OracleModePLParserListener.java

```diff
@@ -555,6 +555,12 @@ public void enterAlter_package_stmt(Alter_package_stmtContext ctx) {
         }
     }
 
+    @Override
+    public void enterCall_spec(PLParser.Call_specContext ctx) {
+        this.sqlType = sqlType.CALL;
+        this.dbObjectType = DBObjectType.OTHERS;
```

Comment:
same as mysql listener

Reply:
the statement like call functionName() also enters this method, so use OTHERS is better